server/views/probationPractitionerReferrals/dashboard.njk

```diff
@@ -17,6 +17,8 @@
       <h1 class="govuk-heading-xl">{{ presenter.title }}</h1>
 
         {{ mojSubNavigation(subNavArgs) }}
+        {% include "../partials/pagination.njk" %}
+        <br>
```

Comment:
I feel like this isn't the 'proper' way to add the spacing needed

Reply:
👋 @rwk-moj could you help please?